src/test/java/seedu/address/model/affiliation/AffiliationModifierTest.java

```diff
@@ -78,4 +78,23 @@ public void nameChangeAffiliation_validParam_success() {
                 patientB.getName(), newName, model);
         assertTrue(doctorA.getAffiliations().contains(new Affiliation(newName.fullName)));
     }
+
+    @Test
+    public void nameChangeAffiliationHistory_validParam_success() {
+        Person doctorAlice = new DoctorBuilder().withName("Alice")
+            .withAffiliationHistory("Bob").build();
+        Person doctorCharlie = new DoctorBuilder().withName("Charlie")
+            .withAffiliationHistory("Bob").build();
+        Person patientBob = new PatientBuilder().withName("Bob")
+            .withAffiliations("Alice", "Charlie")
+            .withAffiliationHistory("Alice", "Charlie").build();
+        model.addPerson(doctorAlice);
+        model.addPerson(doctorCharlie);
+        model.addPerson(patientBob);
```

Comment:
Would it be possible to add another `Doctor` into the model, where the doctor is in the `AffiliationHistory` of Bob, but itself has an empty `AffiliationHistory`? I understand this is an impossible scenario, but we will be able to hit the test coverage for line #L132 in `EditCommand` with this.

Reply:
Sure, will do!